specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/Jobs.json

```
@@ -1075,12 +1075,75 @@
             "template": {
               "description": "Job's execution container.",
               "$ref": "#/definitions/JobExecutionTemplate"
+            },
+            "detailedStatus": {
+              "description": "Detailed status of the job execution.",
+              "$ref": "#/definitions/ExecutionStatus"
             }
           },
           "x-ms-client-flatten": true
         }
       }
     },
+    "ExecutionStatus": {
+      "description": "Container Apps Job execution status.",
+      "type": "object",
+      "properties": {
+        "pods": {
+          "description": "Pods in the execution.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PodExecutionStatus"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        }
+      }
+    },
+    "PodExecutionStatus": {
+      "description": "Container Apps Job execution pod status.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Pod Name.",
+          "type": "string"
+        },
+        "containers": {
+          "description": "Containers in the execution",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerExecutionStatus"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        }
+      }
+    },
+    "ContainerExecutionStatus": {
+      "description": "Container Apps Job execution container status. Contains status code and reason",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Container Name.",
+          "type": "string"
+        },
+        "code": {
+          "description": "Exit code",
+          "type": "integer",
+          "format": "int32"
+        },
+        "reason": {
+          "description": "Container reason",
```

```suggestion
          "details": "Container additional information details",
```

Since we will change the `reason` field name to `additionalInformation` let's also change `description` to `details` to make it more generic

---

`description` is a property from azure rest api. Not something we specify.